sources/models/src/aws-ecs-1/mod.rs

```diff
@@ -6,7 +6,7 @@ use crate::modeled_types::Identifier;
 use crate::{
     AutoScalingSettings, AwsSettings, BootstrapContainer, CloudFormationSettings, DnsSettings,
     ECSSettings, HostContainer, KernelSettings, MetricsSettings, NetworkSettings, NtpSettings,
-    OciHooks, PemCertificate, RegistrySettings, UpdatesSettings,
+    OciDefaults, OciHooks, PemCertificate, RegistrySettings, UpdatesSettings,
 };
```

Comment:
This PR should get rebased onto develop so we can add `aws-ecs-2` and `aws-ecs-2-nvidia` also.

Reply:
Rebased and added and tested the OCI defaults for aws-ecs-2